go/chat/boxer.go

```diff
@@ -37,9 +38,11 @@ func init() {
 type Boxer struct {
 	utils.DebugLabeler
 
-	tlf    keybase1.TlfInterface
-	hashV1 func(data []byte) chat1.Hash
-	sign   func(msg []byte, kp libkb.NaclSigningKeyPair, prefix libkb.SignaturePrefix) (chat1.SignatureInfo, error) // replaceable for testing
+	tlf                       keybase1.TlfInterface
+	hashV1                    func(data []byte) chat1.Hash
+	sign                      func(msg []byte, kp libkb.NaclSigningKeyPair, prefix libkb.SignaturePrefix) (chat1.SignatureInfo, error) // replaceable for testing
+	testingValidSenderKey     func(context.Context, gregor1.UID, []byte, gregor1.Time) (found, validAtCTime bool, revoked *gregor1.Time, unboxingErr UnboxingError)
+	testingGetSenderInfoLocal func(context.Context, gregor1.UID, gregor1.DeviceID) (senderUsername string, senderDeviceName string, senderDeviceType string)
 	libkb.Contextified
```

Comment:
Can you move this to the top? I like to have the included type at the top to make it easier to read.